app.js

```diff
@@ -15,12 +15,12 @@ dotenv.config();
 
 
 
-
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(openApiDocumentation));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(helmet());
 app.use('/v1', router);
-app.use('/', swaggerUi.serve, swaggerUi.setup(openApiDocumentation));
```

Comment:
Bro, with this the base url returns an error. Remember we need to serve the swagger docs here or at least redirect to the swagger docs.